app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -33,7 +33,7 @@
       android:onClick="@{(v) -> viewModel.onExplorationClicked()}"
       app:cardCornerRadius="4dp"
       app:cardElevation="@dimen/story_chapter_card_playable_state_elevation"
```

Comment:
Use `component_color_story_activity_container_background_color` directly inside `MaterialCardView`

Reply:
Done.

Reply:
You can remove `component_color_story_activity_container_background_color ` from constraintLayout as it is getting used in card directly.

Also, use `app:cardBackgroundColor` for card and remove `android:background`

Reply:
Done.